packages/admin/src/components/pageRouting/pageComponents/GenericPage.tsx

```diff
@@ -20,6 +20,7 @@ export type GenericPageProps =
  * ```
  *
  * @group Pages
+ * @deprecated No longer used
```

Comment:
we should either deprecate all pages, or none. I'm for none for now.

Reply:
Done